sw/device/tests/BUILD

```diff
@@ -3219,55 +3219,6 @@ opentitan_test(
     ],
 )
 
-# This test reproduces a silicon failure, but passes in all other environments.
-# TODO(lowrisc/opentitan#20798)  Remove this test and extend
-# pwrmgr_random_sleep_all_wake_ups to cover all wakeups once this issue is
-# fixed.
-opentitan_test(
```

Comment:
Don't we want to keep the test to the test Silicon and also to make sure that the bug will never show up again?

Reply:
I see, thanks for explaining.